nifi-frontend/src/main/frontend/apps/nifi-registry/src/app/app.component.html

```diff
@@ -15,9 +15,4 @@
   ~ limitations under the License.
   -->
 
-<div class="flex flex-col h-screen">
-    <header class="mb-5 nifi-registry-header">
-        <app-header></app-header>
-    </header>
-    <router-outlet></router-outlet>
-</div>
+<router-outlet></router-outlet>
```

Comment:
Removing the app-header from here has also, in effect, removed it from the buckets page since it wasn't added there specifically along with this change.